src/index.html

```diff
@@ -7,8 +7,96 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com"
+    >
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="true"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img 
+          class="logo"
+          src="images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="header nav-list">
+          <li class="header nav-item">
+            <a 
+              href="#home" 
+              class="link is-active default-blue"
+            >Apple
+            </a>
+          </li>
+
+          <li class="header nav-item">
```

Comment:
you dont need `header` class here 
```suggestion
          <li class="nav__item">
```